ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/management-security.md

```diff
@@ -109,4 +111,9 @@
           certificate:
             file: <str>
             key: <str>
+
+          # List of CRLs (Certificate Revocation List).
+          # One CRL needs to be specified for every CA in the chain, even if it's not revoking any certificate.
```

Comment:
I am not sure this is true. The revocation lists are not required on EOS are they?

Reply:
This is documented here - https://www.arista.com/en/um-eos/eos-control-plane-security

grep for 'One CRL needs to be specified for every CA'

Reply:
Yes the crl field itself is optional. but if it is specified, one for each cert in the chain needs to be specified. Let me rephrase this a bit.

Reply:
You can use min_length on the list in the schema to enforce this.

Reply:
nvm I misunderstood. Resolving.